### ISSUE TITLE
Allow enemies MUCH LESS ACCURACY at longer ranges

### ISSUE THREAD

**Issue by LeeBamberTGC** [TheGameCreators/GameGuruRepo#2433]
Like so:
![image](https://user-images.githubusercontent.com/35840583/157076570-4390187b-1d13-4b7a-8314-264b8fe9ad20.png)

**Comment by ZakJudges** [comment]
I feel it would be nice to have settings like these made available to the end user. That way we can tweak the difficulty of different characters. For example, you could make different classes of enemy for your game... a close-quarters combatant that is deadly at close range but not great at distance, or vice versa.
This would also perhaps facilitate some global difficulty system for the game as a whole, but in my opinion, the main benefit is variety of combat gameplay, preventing every enemy from feeling the same :)

**Comment by LeeBamberTGC** [comment]
@ZakJudges Yes I think this needs more thought. I have bumped to post EA so we can do it 'properly' :)
